docs/api/checkouts.rst

```diff
@@ -1,235 +1,560 @@
+            "hostname":"localhost",
+            "color":"MediumPurple",
+            "created_at":"2017-02-14T11:10:02+0100",
+            "updated_at":"2017-02-14T11:10:02+0100",
+            "enabled":true,
+            "tax_calculation_strategy":"order_items_based",
+            "_links":{
+                "self":{
+                    "href":"\/api\/v1\/channels\/1"
                 }
-            },
-            "code": "WEB-US",
-            "color": "Pink",
-            "created_at": "2014-12-03T09:54:28+0000",
-            "enabled": true,
-            "id": 3,
-            "name": "United States Webstore",
-            "type": "web",
-            "updated_at": "2014-12-03T09:58:29+0000"
+            }
+        },
+        "shipping_address":{
+            "first_name":"Elon",
+            "last_name":"Musk",
+            "country_code":"US",
+            "street":"10941 Savona Rd",
+            "city":"\u2019Los Angeles",
+            "postcode":"CA 90077"
         },
-        "checkout_state": "addressing",
-        "comments": [],
-        "confirmed": true,
-        "created_at": "2014-12-15T13:15:22+0000",
-        "email": "xschaefer@example.com",
-        "expires_at": "2014-12-15T16:15:22+0000",
-        "id": 52,
-        "items": [],
-        "items_total": 1500000,
-        "payments": [],
-        "shipments": [],
-        "state": "cart",
-        "total": 1499750,
-        "updated_at": "2014-12-15T13:37:29+0000",
-        "user": {
-            "credentials_expired": false,
-            "email": "xschaefer@example.com",
-            "email_canonical": "xschaefer@example.com",
-            "enabled": true,
-            "expired": false,
-            "group": [],
-            "id": 5,
-            "locked": false,
-            "roles": [],
-            "username": "xschaefer@example.com",
-            "username_canonical": "xschaefer@example.com"
-        }
+        "billing_address":{
+            "first_name":"Elon",
+            "last_name":"Musk",
+            "country_code":"US",
+            "street":"10941 Savona Rd",
+            "city":"\u2019Los Angeles",
+            "postcode":"CA 90077"
+        },
+        "payments":[
+            {
+                "id":21,
+                "method":{
+                    "id":1,
+                    "code":"cash_on_delivery"
+                },
+                "amount":108787,
+                "state":"cart"
+            }
+        ],
+        "shipments":[
+            {
+                "id":21,
+                "state":"cart",
+                "method":{
+                    "code":"ups",
+                    "enabled":true
+                }
+            }
+        ],
+        "currency_code":"USD",
+        "locale_code":"en_US",
+        "checkout_state":"addressed"
     }
 
+Of course you can specify different shipping and billing address. If Elon would like to to send a present to NASA administrator, Frederick D. Gregory, he could send the following request:
```

Comment:
`Of course, you can specify different shipping and billing addresses. If our user Elon would like to send a gift to the NASA administrator, Frederick D. Gregory, he could send the following request:`